api/models.py

```diff
@@ -109,7 +109,16 @@ def update(self):
         self.updated = datetime.utcnow()
 
     @classmethod
-    def modify_parent(cls, query_param_dict):
-        """Modify query parameter dictionary for parent"""
-        query_param_dict['parent'] = ObjectId(query_param_dict['parent'])
-        return query_param_dict
+    def translate_fields(cls, params: dict):
+        """Translate fields in `params` into objects as applicable
+
+        Translate fields reprensted by strings in the `params` dictionary into
```

Comment:
Need to fix a minor typo.
```suggestion
        Translate fields represented by strings in the `params` dictionary into
```